packages/react-saasify/src/components/LiveServiceDemo/LiveServiceDemo.js

```diff
@@ -196,35 +227,55 @@ export class LiveServiceDemo extends Component {
   }
 
   _onClickRun = () => {
-    if (this.state.output) {
-      this.setState({
-        output: null,
-        outputContentType: null
-      })
+    const { auth, service } = this.props
```

Comment:
Definitely useful to externalise, but why as a component and not as a lib/helper? Alternatively, we could move it into `ServiceForm/helpers.js`

Reply:
I think we'll likely want to have this as a top-level component and I kind of like the name `ServicePlayground`. Would also be fine to see this logic moved into `ServiceForm`.

Just want to make sure we're not putting too much into `LiveServiceDemo` since I could potentially see us having the playground aspect of things live in more places in the future.